workshops/Tkinter/README.md

```diff
@@ -0,0 +1,135 @@
+This line is big dont worry its also simple. Here we need a button to submit it right?
+For that lets call the Button() function. Inside this function we need to specify the window aswell, Then we need another function command which means same as onClick is js . This does the action when we click on the button. Actually it calls the function Display which is described in the lines L2 and L3. The next keyword is text you know what it is. Then font you know that too, Then comes the width and height here you can give the width and height of the button as integers. Then as usual we need to place it using place() function.
+
+### L12
+
+<img width="963" alt="mainloop" src="https://user-images.githubusercontent.com/54895485/93660596-535a6100-fa6e-11ea-81c0-9ed48cbb4559.png">
+
+mainloop() is an infinite loop used to run the application, wait for an event to occur and process the event as long as the window is not closed.
+This is a loop so it needs to be repeated until we close the window thats it .
+
+### L2
+
+<img width="959" alt="Screenshot 2020-09-19 at 11 51 33 AM" src="https://user-images.githubusercontent.com/54895485/93660618-83a1ff80-fa6e-11ea-947a-4afd6c307e05.png">
+This is the function which executes when we press the submit button because we specified command = this function.
+
+### L3
+
+<img width="959" alt="Screenshot 2020-09-19 at 11 51 47 AM" src="https://user-images.githubusercontent.com/54895485/93660624-90beee80-fa6e-11ea-9bf7-327411e77ab4.png">
+ 
+Now we need to print what we typed right? .For now lets print it in the terminal/CMD so we use print(then we need to get the value of w variable we defined inorder to collect the typing data.) for that we use get function.
+![ezgif com-gif-to-mp4](https://user-images.githubusercontent.com/54895485/93177183-087bd900-f750-11ea-8df0-966f196b88cb.gif)
+## Now you have made the simple UI.
+
+Here is the challenge for you .WE MADE THE UI AND PRINTED THE TYPINGS IN TERMINAL .YOU NEED TO PRINT IT IN THE SCREEN ITSELF.
+
+## Step 4: Your Custom Part
+
+Now you know Tkinter a little bit . You should not stop here inorder to learn more and master in UIs you need to practice a lot. The best way to learn it is to look at the world and think of an idea then try to make it. You will learn Tkinter soon because its simple .
+
+## Lastly dont stuck here i will come up with more workshops to make you know that tkinter is actually cool.
+I am attaching the Tkinter documentation take a look [Learn Tkinter](https://docs.python.org/3/library/tk.html).
+Also if you are new to python take a look at [python](https://www.python.org/doc/) documentation too 
+### Softwares made using Tkinter by other hackers.
+
+[Hariprasad rajan](https://repl.it/@HariprasadR03/hackthon#main.py)
+[Anupriya shaji](https://repl.it/@Anupriya567/qtshb#main.py)
+[kk Haridev](https://repl.it/@DandaThor/h#main.py)
+[Bezlin Johnson](https://repl.it/@bezlin/graphics)
+
+## Big platforms made using Tkinter
+
+Billing softwares
```

Comment:
provide links

Reply:
ok

Reply:
I will do that I have note down the changes and update it